### DISCUSSION TITLE
Welcome to U08M11002 Discussions!

### DISCUSSION THREAD

**Discussion by aprilabdotdev** [aprilabdotdev/U08M11002#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wushenhaoyu** [comment]
虽然说我考得不太好，但是老师的信号题救了我的命
